src/quantuminspire/qiskit/backend_qx.py

```diff
@@ -308,58 +305,82 @@ def __qubit_to_classical_hex(qubit_register, measurements, number_of_qubits):
         return classical_state_hex
 
     @staticmethod
-    def __get_memory_data(raw_data, measurements, number_of_qubits):
-        """ The quantum inspire backend returns the single shot values. This function
-            converts the raw data to hexadecimal memory data according the Qiskit spec.
-
-        Note:
-            When shots = 1, the backend returns an empty list as raw_data.
+    def __convert_histogram(result: Dict, measurements: Dict) -> Obj:
```

Comment:
Guess the comments above, or the questions, apply to all container hints.

Reply:
Added type hint for Dict